modules/ROOT/pages/vpc-provisioning-concept.adoc

```diff
@@ -5,62 +5,66 @@ endif::[]
 
 Provisioning a VPC for your organization consists of two major steps:
 
-. Creating the VPC. +
+. Create the VPC. +
 This is a self-service process. You can create a VPC using either the Runtime Manager UI, or the Anypoint Platform CLI.
-. Connecting to your VPC. +
-This needs to be done by a MuleSoft representative.
+. Connect to your VPC. +
+This must be done by a MuleSoft representative.
 
-See the tutorials and task in the See Also section below for detailed information about these processes. +
-However, before creating the VPC, you need to be aware that some settings are only configurable during the first set up:
+[NOTE]
+--
+To avoid unnecessary downtime, it is critical to define the VPC settings during the initial setup of the VPC. +
```

Comment:
does "initial setup" = "Create the VPC" step? Any way to make this clear (by mentioning "setup" or "configure" in the description of that step?

Reply:
I put ...when you first create a new VPC.
Create VPC is the terminology used in the UI.